site/_docs/reference.md

```diff
@@ -2758,6 +2758,7 @@ The 'C' (compatibility) column contains value:
 * 'p' for PostgreSQL ('fun=postgresql' in the connect string),
 * 'r' for Amazon RedShift ('fun=redshift' in the connect string),
 * 's' for Apache Spark ('fun=spark' in the connect string).
+* 'i' for ClickHouse('fun=clickhouse' in the connect string).
```

Comment:
Ask a question not related to PR, why is there only one letter to represent the dialect here? It feels very abstract.

Reply:
As I know it is a doc standard and there a document checker for it @xiedeyantu

Reply:
If you see the reference page, each function has a list of dialects:
https://calcite.apache.org/docs/reference.html#dialect-specific-operators
I suppose the intent is to keep that list shorter so it can be displayed nicely.